### PR TITLE
fix(kubernetes): [CKV_K8S_140] Update ApiServerTlsCertAndKey.py to check RHS values 

### PR DESCRIPTION
**Pull request by nijdarshan**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description

The earlier code was only checking if the `-tls-cert-file` and `--tls-private-key-file` parameters are present but was not checking if the values are provided on the RHS.

### Fixes 
Optimised `CKV_K8S_140` to check for the values and non-empty values on the RHS of the parameters. 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by gruebel — COMMENTED** [review]
nice catch 🚀

File: checkov/kubernetes/checks/resource/k8s/ApiServerTlsCertAndKey.py [hunk 1]
```diff
@@ -18,9 +18,11 @@ def scan_container_conf(self, metadata: Dict[str, Any], conf: Dict[str, Any]) ->
                 hasKeyCommand = False
                 for command in conf["command"]:
                     if command.startswith("--tls-cert-file"):
-                        hasCertCommand = True
+                        if len(command.split("=")) == 2 and (command.split("=")[1]).strip() != '':
```
[the remark this answers is not in the document]
Comment (nijdarshan) [reply checkov/kubernetes/checks/resource/k8s/ApiServerTlsCertAndKey.py:21]: the second time I'm making sure there isn't an empty string after the equals to sign

[the remark this answers is not in the document]
Comment (nijdarshan) [reply checkov/kubernetes/checks/resource/k8s/ApiServerTlsCertAndKey.py:21]: what I mean to say is the check having the first split would still pass even if there is no value but just a space

[the remark this answers is not in the document]
Comment (gruebel) [reply checkov/kubernetes/checks/resource/k8s/ApiServerTlsCertAndKey.py:21]: I understood that 🙂 if you just want to check, if a string is empty or not, then you can leverage the truthiness of it in Python. So these statements do the same, but latter one is the preferred and more performant one.
```python
cert_file[1].strip() != ''
``` 
```python
cert_file[1].strip()
```

[the remark this answers is not in the document]
Comment (nijdarshan) [reply checkov/kubernetes/checks/resource/k8s/ApiServerTlsCertAndKey.py:21]: Oh wow! I didn't know this Anton; I just tested locally and will make the changes now

[the remark this answers is not in the document]
Comment (nijdarshan) [reply checkov/kubernetes/checks/resource/k8s/ApiServerTlsCertAndKey.py]: Updated this, please check. And I intend to optimise the rest of the checks as well.

**Review by gruebel — APPROVED** [review]
nice, looks good 🚀  can you also add one failing test, where one of the flags is used, but without a value?

**Comment by nijdarshan** [comment]
Added a new fail check for the update

**Comment by nijdarshan** [comment]
> nice, looks good 🚀 can you also add one failing test, where one of the flags is used, but without a value?

Hey, @gruebel could help me identify why the code fails in a few tests? I will rectify them accordingly!

**Comment by gruebel** [comment]
quite easy, if you add a new test case it also needs to be added to the actual test file. Additionally the object names should be unique, so we can be sure, the correct ones are failing/passing.

**Review by Saarett — APPROVED** [review]

**Pull request merged**